README.md

```diff
@@ -16,13 +16,17 @@ and our methods reflect this differentiation.
 
 ---
 
-<!-- ## Usage
+## Usage
+
+
+For a demonstration of how to use the package, see https://colab.research.google.com/drive/1Hd7dQ93GO1shCMbKORwpiG0MEsj7dtC3?usp=sharing
```

Comment:
you should download the ipynb file and add it to the repo.
then, create an `open in colab` badge https://openincolab.com/